pipeline/app.py

```diff
@@ -35,13 +35,27 @@ def insert_user_data(conn: connection, data_user: dict):
     Inserts user data into users table in required database.
     """
     cur = conn.cursor(cursor_factory=extras.RealDictCursor)
-    query = "INSERT INTO users(email, first_name, last_name) VALUES (%s, %s, %s)"
-    cur.execute(query, (data_user["email"],
-                        data_user["first_name"],
-                        data_user["last_name"]))
 
-    conn.commit()
-    cur.close()
+    selection_query = "SELECT email FROM users;"
+
+    cur.execute(selection_query)
+    rows = cur.fetchall()
+
+    emails = [row["email"] for row in rows]
+    print(emails)
```

Comment:
You may still be using this print statement in testing but just to point it out so you know its still here :)